modules/k3s/k3s.go

```diff
@@ -163,3 +164,21 @@ func unmarshal(bytes []byte) (*KubeConfigValue, error) {
 	}
 	return &kubeConfig, nil
 }
+
+// LoadImages loads images into the k3s container.
+func (c *K3sContainer) LoadImages(ctx context.Context, images string) error {
```

Comment:
Do we need to validate it's a valid tar file or, on the contrary, it's fine and enough to fail in https://github.com/testcontainers/testcontainers-go/pull/1622/files#diff-85b0f02d7304331291be9d2df1c8a91dc7afd3a7b0bd2d0ffe3afd686a8a99b8R178?

Reply:
This is validated by docker. I think we only need to propagate docker's errors here.

Reply:
We probably need to update the imports for path and filepath :)